config/runtime_flags.go

```diff
@@ -2,8 +2,13 @@ package config
 
 import "golang.org/x/exp/maps"
 
-// RuntimeFlagGRPCDatabrokerKeepalive enables gRPC keepalive to the databroker service
-var RuntimeFlagGRPCDatabrokerKeepalive = runtimeFlag("grpc_databroker_keepalive", false)
+var (
+	// RuntimeFlagGRPCDatabrokerKeepalive enables gRPC keepalive to the databroker service
+	RuntimeFlagGRPCDatabrokerKeepalive = runtimeFlag("grpc_databroker_keepalive", false)
+
+	// RuntimeFlagStripFromPort enables stripping the port from incoming requests for matching from addresses
+	RuntimeFlagStripFromPort = runtimeFlag("strip_from_port", true)
```

Comment:
One comment on the name: with this implementation it doesn't really "strip" the port, does it? I think maybe something along the lines of "match_any_incoming_port" might be more accurate.